library/neutron_network.py

```diff
@@ -18,6 +18,8 @@
 
 try:
     from neutronclient.neutron import client
+    from keystoneauth1.identity import v2
```

Comment:
So this ansible module doesn't work with v3. I'm changing only the code necessary to keep using the v2 API, as previously it was using v2 just by nature of the endpoint we had in the catalog. Eventually we'll move away from this module and use one of the `os_whatever` modules instead, which all use shade and support v3.